src/module/comment.js

```diff
@@ -0,0 +1,167 @@
+import renderError from './error.js';
+
+const makeComment = async (username, userComment, id) => {
+  const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/DG1557loKgI2XXAUI0g2/comments/';
+  try {
+    username = username.value;
+    userComment = userComment.value;
+    if (username !== '' && userComment !== '') {
+      const newComment = {
+        item_id: id,
+        username,
+        comment: userComment,
+      };
+
+      await fetch(baseUrl, {
+        method: 'POST',
+        headers: {
+          'Content-type': 'application/json',
+        },
+        body: JSON.stringify(newComment),
+      });
+    }
+    username = '';
+    userComment = '';
+  } catch (err) {
+    renderError(err.message);
+  }
+};
+
+const showAllComments = async (mealId) => {
+  const baseUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/DG1557loKgI2XXAUI0g2/comments?item_id=${mealId}`;
+  let data;
+
+  try {
+    const request = await fetch(baseUrl);
+    const response = await request.json();
+    data = await response;
+  } catch (err) {
+    renderError(err.message);
+  }
+  return data;
+};
+
+const commentPopUp = async (meal) => {
+  const mealCard = document.createElement('div');
+  const mealImg = document.createElement('img');
+  const mealTitle = document.createElement('h3');
+  // const mealIgredient = document.createElement('div'); // Add meal ingredient (v2.0)
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.